File: java/src/algorithm/boj/sort/BOJ2751.java
package algorithm.boj.sort;

import java.io.*;
import java.util.Arrays;

public class BOJ2751 {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
        int n = Integer.parseInt(br.readLine());
        String[] arr = new String[n];
        for (int i = 0; i < n; i++) {
            arr[i] = br.readLine();
        }
        Arrays.sort(arr, (o1, o2) -> Integer.parseInt(o1)-Integer.parseInt(o2));
        for (String s : arr) {
            bw.write(s + "\n");
        }
        bw.flush();
    }
}

File: ps_kotlin/src/main/kotlin/boj/boj2751.kt
package boj

fun main() {
    val br = System.`in`.bufferedReader()
    val bw = System.out.bufferedWriter()

    val n = br.readLine().toInt()
    val arr = BooleanArray(2000001)
    for (i in 0 until n) {
        arr[br.readLine().toInt() + 1000000] = true
    }
    for ((i, b) in arr.withIndex()) {
        if (b)
            bw.write("${i - 1000000}\n")
    }
    bw.flush()
}